### PR TITLE
frontend: Add favicon and 404 to subdomains

### PR DESCRIPTION
**Pull request by juliaogris**
Add favicon and 404 to subdomains `preview` (soon to be `docs`) and
`discord`.

### CODE REVIEW

File: frontend/preview/img/favicon.png [hunk 1]
```diff
@@ -0,0 +1 @@
+../../img/favicon.png
```
Comment (camh-) [inline frontend/preview/img/favicon.png:1]: Why do some html pages say the favicon is `favicon.ico` and others say it is `img/favicon.png`?

**Review by camh- — APPROVED** [review]
Seems fine. That clean target is getting a bit yucky.

Not sure I understand the need for multiple favicons in different locations - I'm sure there's some webby explanation.

File: frontend/preview/img/favicon.png (hunk 1 above)
Comment (juliaogris) [reply frontend/preview/img/favicon.png:1]: Fixed, using img/favicon.png everywhere now.

Comment (juliaogris) [reply frontend/preview/img/favicon.png:1]: apologies, I'm merging this now and resolving myself. let me know if that offends.

**Pull request merged**